ui/client/reducers/graph/utils.ts

```diff
@@ -151,7 +151,7 @@ export function addNodesWithLayout(state: GraphState, {nodes, layout}: ReturnTyp
 export function removeSubprocessVersionForLastSubprocess(processToDisplay: Process, idToDelete: NodeId) {
   const subprocessVersions = processToDisplay.properties.subprocessVersions
   const nodeToDelete = processToDisplay.nodes.find(n => n.id === idToDelete)
-  if (nodeToDelete.type === "SubprocessInput") {
+  if (nodeToDelete?.type === "SubprocessInput") {
```

Comment:
Can we have node without type? And I think node's type should be enum ;)

Reply:
find may return nothing.
no it shouldn't be enum, brings nothing and possibly infinite list